tests/page_objects/care_platform_page.yaml

```diff
@@ -512,4 +514,19 @@ care_platform_facility_Record:
   info_title: //h4[text()='Facility Information']
   devices_button: //button[text()='Devices']
   device_status: //div[@class='account-status account-status-active']
-  
+
+care_platform_edit_participants_vendor:
+  add_vendor_button: //button[text()='Add Vendor']
+  cancel_vendor_modal_button: //div[@id='modals']//button[text()='Cancel']
+  category_modal_input: //select[@id='category']/option[@value ='$AND_CLI_CATEGORY$']
+  save_close_modal_button: //button[text()='Save and Close']
+  edit_created_vendor_button: //div[@class='flex-col flex-col-9']//div[contains(@class, 'vendor')]//div[text()='$AND_CLI_NAME$']//following-sibling::div[contains(@class,'flex-col')]/button[text()='Edit']
+  name_vendor_record: //div[@class='flex-col flex-col-9']//div[contains(@class, 'vendor')]//div[text()='$AND_CLI_NEW_NAME$']
+  delete_created_vendor_button: //div[@class='flex-col flex-col-9']//div[contains(@class, 'vendor')]//div[text()='$AND_CLI_NEW_NAME$']//following-sibling::div[contains(@class,'flex-col')]/button[text()='Delete']
```

Comment:
As 'flex-col' and 'flex-col flex-col-9' are not reliable classes, and they are not specific of the object (as it is some IDs, or text, or other classes that you have used example: 'tabs-nav') I recommend you not to use it. That class 'flex-col' is more of a style attribute of the column and it could easily change...

Reply:
I removed this attribute for the selectors.